Source/WebKit/UIProcess/mac/WebViewImpl.h

```diff
@@ -229,10 +229,12 @@ class WebViewImpl final : public CanMakeWeakPtr<WebViewImpl>, public CanMakeChec
     ~WebViewImpl();
 
     NSWindow *window();
+    RetainPtr<NSWindow> protectedWindow();
 
     WebPageProxy& page() { return m_page.get(); }
 
     WKWebView *view() const { return m_view.getAutoreleased(); }
+    RetainPtr<WKWebView> protectedView() const { return view(); };
```

Comment:
This should use `return m_view.get();`. Calling `view()` isn't great here because it will unnecessarily autorelease.